Okta.Xamarin/Okta.Xamarin.UITest.Shared/TestOktaConfig.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Okta.Xamarin.UITest
+{
+	// TODO: move OktaConfig to Okta.Xamarin.Shared to eliminate the need for this class
```

Comment:
Is this TODO still valid?

Reply:
Yes.  There are a number of redundant classes that will add to confusion in the future if not consolidated; this class is one.